examples/sample-layers/layers/multi-color-path-layer/multi-color-path-layer.js

```diff
@@ -0,0 +1,51 @@
+import {PathLayer} from 'deck.gl';
+
+// PathLayer subclass by @dcposch
+export default class MultiColorPathLayer extends PathLayer {
```

Comment:
@dcposch FYI - Using your idea as an example for how to subclass layers, as discussed in #336